tests/integration/test_migrate.sh

```diff
@@ -72,7 +72,7 @@ trap cleanup EXIT
 # at /var/lib/docker
 atomic_migrate () {
 	setup
-	${ATOMIC} migrate export --dir "$(pwd)/migrate-dir"
+	echo 'y'|${ATOMIC} migrate export --dir "$(pwd)/migrate-dir"
```

Comment:
Ahhh, sounds good. I was thinking of the yum/dnf `-y` switch, but apparently the long opt for it is `--assumeyes`, not `--yes` as I thought.